app/models/reservation.rb

```diff
@@ -118,9 +120,15 @@ def self.relay_in_progress
     where("actual_start_at IS NOT NULL AND actual_end_at IS NULL")
   end
 
+  scope :not_offline, -> { where(type: nil) }
```

Comment:
They unfortunately aren't. The base `Reservation` gets a type of `nil` by default:
```
[1] pry(main)> Instrument.first.reservations.create.type
  Instrument Load (0.4ms)  SELECT  `products`.* FROM `products` WHERE `products`.`type` IN ('Instrument')  ORDER BY `products`.`id` ASC LIMIT 1
   (0.1ms)  SAVEPOINT active_record_1
  Product Load (0.4ms)  SELECT  `products`.* FROM `products` WHERE `products`.`id` = 3 LIMIT 1
   (0.1ms)  RELEASE SAVEPOINT active_record_1
=> nil
```